.travis.yml

```diff
@@ -98,3 +102,7 @@ after_script:
     - echo "Running tests in SPC using ./hack/run_build.sh"
     - "./hack/run_build.sh"
 
+after_success:
+    - bash <(curl -s https://codecov.io/bash) -f coverage.txt
```

Comment:
This pattern makes me nervous for automated use, but not for the typical security-related reason.  Since execution in a public/cloud context, far away from sensitive data/infrastructure, we can somewhat ignore the remote possibility of a rogue script.  The more immediate issue is this adds a dependency between PR testing and an external service/resources.  Testing/automation should be mostly focused stability, but the upstream script source is more focused on features.  So my suggestion is to download the script once and add it to this repository for execution.  Later if it needs to change or get updated, that can be validated manually as a dedicated change.